src/main/java/com/ita/if103java/ims/dao/impl/DashboardDaoImpl.java

```diff
@@ -162,43 +174,62 @@ public void fillList(WarehousePremiumStructDto wpld, List<WarehousePremiumStruct
                 "ORDER BY sum(ts.quantity) " +
                 "LIMIT ? ";
 
-        static final String SQL_FIND_WAREHOUSE_LOAD =
-            "SELECT wh.top_warehouse_id, sum(quantity*volume) AS charge, sum(wh.capacity) AS capacity " +
-                "FROM saved_items si " +
-                "JOIN items it " +
-                "ON si.item_id=it.id " +
-                "JOIN warehouses wh " +
-                "ON si.warehouse_id=wh.id " +
-                "WHERE wh.is_bottom=1 " +
-                "GROUP BY wh.top_warehouse_id";
-
-        static final String SQL_FIND_ENDED_ITEMS =
+        static final String SQL_FIND_WAREHOUSE_LOAD_BY_ACCOUNT_ID =
+            "SELECT wcap.id, ifnull(charge,0) charge, capacity " +
+                "FROM " +
+                "(SELECT wh.top_warehouse_id id, sum(capacity) capacity, account_id, active " +
+                "FROM warehouses wh WHERE account_id=? and active=1 GROUP BY wh.top_warehouse_id) AS wcap " +
+                "LEFT OUTER JOIN " +
+                "(SELECT top_warehouse_id id, sum(quantity*volume) charge " +
+                "FROM saved_items si, items it, warehouses wh " +
+                "WHERE si.item_id=it.id and si.warehouse_id=wh.id " +
+                "GROUP BY top_warehouse_id) AS wcharge " +
+                "ON wcap.id=wcharge.id ";
```

Comment:
Why did you do so drastically changes to what already worked? Please explain me in ITA